Include/internal/pycore_ceval.h

```diff
@@ -138,12 +138,12 @@ extern void _PyEval_DeactivateOpCache(void);
 /* With USE_STACKCHECK macro defined, trigger stack checks in
    _Py_CheckRecursiveCall() on every 64th call to _Py_EnterRecursiveCall. */
 static inline int _Py_MakeRecCheck(PyThreadState *tstate)  {
-    return (tstate->c_recursion_remaining-- <= 0
+    return (tstate->c_recursion_remaining-- < 0
```

Comment:
The reason this change is here is to make the guard `tstate->c_recursion_remaining <= 0` instead of `tstate->c_recursion_remaining <= 1`. Comparing to zero is generally faster.

It also seems more correct. If `c_recursion_remaining` is 1, that suggest that there is 1 remaining call, not 0.

Reply:
The actual number we compare to doesn't really matter, as long as we are consistent.